mlrun/runtimes/utils.py

```diff
@@ -106,6 +107,14 @@ def resolve_mpijob_crd_version(api_context=False):
     return cached_mpijob_crd_version
 
 
+def resolve_spark_operator_version():
+    try:
+        regex = re.compile("spark-([23])")
+        return int(regex.findall(config.spark_operator_version)[0])
+    except Exception:
+        raise ValueError("Failed to resolve spark operator's version")
```

Comment:
Why not just making the values `2` or `3` ?

Reply:
Because helm fails to install the chart when passing an integer (somehow passed from provazio as int and not str)